specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/integrationRuntime.json

```diff
@@ -382,7 +382,7 @@
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/getObjectMetadata": {
       "post": {
-        "operationId": "IntegrationRuntimeObjectMetadata_Get",
+        "operationId": "IntegrationRuntimeObjectMetadata_List",
```

Comment:
>IntegrationRuntimeObjectMetadata_List [](start = 24, length = 37)

Why do you change the operationId?

Reply:
For the error `ListInOperationName` reported by autorest. [Link](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r1003-listinoperationname)

Reply:
Makes sense